.github/workflows/ai-dispatch.yml

```
@@ -0,0 +1,147 @@
+          CMD:   ${{ steps.intent.outputs.command }}
+          TGT:   ${{ steps.intent.outputs.target }}
+          NOTES: ${{ steps.intent.outputs.notes }}
+        run: |
+          set -eu
+          : "${CMD:?command missing}"
+          gh issue comment "$ISSUE" \
+            --repo "${{ github.repository }}" \
+            --body "🤖 AI dispatch queued: \`$CMD\` (target: \`$TGT\`).
+Claude will pick this up shortly. Notes: ${NOTES:-(none)}.
+
+_This repo is maintained in an AI-native workflow — see README._"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)